contributors/devel/sig-testing/monitoring.md

```diff
@@ -0,0 +1,212 @@
+tests.
+Going from top to bottom, you will see
+* date
+* time
+* job run ID
+* Kubernetes commit that was tested
+* (Most often) test-infra commit
+
+4. **Reason for failure**
+
+The aim for this issues is to begin investigating - you don't have to find the
+reason for failure right away (nor the solution).
+However, do post any information you find usesful.
+
+One way of getting useful information is to click on the failed runs (the red
+rectablges).
+This will send you to a page called **spyglass**.
+
+If we do this for the above test failures in `node-kubelet-features-master`, we
+will see the following
+
+<img src="./testgrid-images/spyglass-sumary.png" height="60%" width="100%">
+
+Right away it will show you what tests failed.
+Here we see that 2 test failed (both related to the node problem detector) and
+the `e2e.go: Node Tests` stage was marked as failed (because the node problem
+detector tests failed).
+
+You will often see "stages" (steps in an e2e job) as mixed with the tests
+themselves.
+The statges imform you of what is going on in the e2e job when an error
+occurred.
+
+If we click on the first test error we will see logs that will (hopefully) help
+us figure out why the test failed.
+
+<img src="./testgrid-images/spyglass-result.png" height="60%" width="100%">
+
+Further down the page you will see all the logs for the entire test run.
+Please copy any information you think may be useful from here into the issue.
+
+5. **Anything else we need to know**
+
+There is this wonderful page built by SIG testing that often comes in handy:
+https://storage.googleapis.com/k8s-gubernator/triage/index.html?pr=1
+This page is called **Triage**.
+We can use it to see if test we see failing in a given job has been failing in
+others and, in general, to understand how jobs are behaving.
+
+For example, we can see how the job we have been looking at has been behaving
+recently.
+
+There is one important detail we have to mention at this point, the job names
+you see on TestGrid are often aliases.
+For example, when we clicked on a test run for
+`node-kubelet-features-master`
+(
+https://testgrid.k8s.io/sig-release-master-informing#node-kubelet-features-master
+), at the top left corner of spyglass the page tells us the real job name,
+`ci-kubernetes-node-kubelet-features` (notice the "ci-kubernetes-" prefix).
+Then we can use this full job name in Triage
+
+https://storage.googleapis.com/k8s-gubernator/triage/index.html?pr=1&job=ci-kubernetes-node-kubelet-features
+
+At the time of this writing we saw the following
+
+<img src="./testgrid-images/triage.png" height="50%" width="100%">
+
+**Note**: notice that you can also improve your query by filtering or excluding
+results based on test name or failure text.
+
+Sometime Triage will help you find patterns to figure out whats wrong.
+In this instance we can also see that this job has been failing rather
+frequently (about 2 times per hour).
+
```

Comment:
Unsure if this is out of scope for this issue or whether it falls more under "revamping the issue template"? -> We should also mention adding a relevant SIG through `/sig <name>` and/or cc'ing relevant people like `/cc @kubernetes/sig-<foo>-test-failures` or `@kubernetes/ci-signal`